snapcraft_legacy/ruff.toml

```
@@ -0,0 +1,30 @@
+# ruff configuration file to run on legacy code and tests.
+# generated with flake8-to-ruff
+exclude = [
+    ".direnv",
+    ".git",
+    ".hg",
+    ".mypy_cache",
+    ".tox",
+    ".venv",
+    ".vscode",
+    "_build",
+    "buck-out",
+    "__pycache__",
+    "build",
+    "dist",
+    "parts",
+    "stage",
+    "prime",
+]
```

Any reason to do this rather than use `extend-exclude` and rely on the [default configuration](https://github.com/charliermarsh/ruff#configuration)?

```suggestion
extend-exclude = [
    ".vscode",
    "__pycache__",
    "parts",
    "stage",
    "prime",
]
```

Not much of a difference, just curious.

---

no reason! This file was generated with `flake8-to-ruff`. Your way makes more sense, so updated
thanks

---

Oh right - `flake8-to-ruff` would make it explicit so you don't have subtle changes. That's smart.